### DISCUSSION TITLE
If I think there is an issue with description in the thriveopedia wiki, where do I report that?

### DISCUSSION THREAD

**Discussion by temujin-borjin** [Revolutionary-Games/Thrive#5257] [Q&A]
I noticed that in the wiki description of the thermal synthesis organelles it says the prokaryote part (thermosynthase) is more efficient than the corresponding eukaryotic part (thermoplast) and vice versa, and, while I'm not sure if it's intentional, later in the `strategy` part of both descriptions it swaps their reported efficiency around with thermoplast being more efficient than thermosynthase.

**Comment by CI09** [comment]
You mean it says thermosynthase is more efficient than the other or...? Also you can create issue on Issues tab in right upper corner. You don't have to use the template.

**Comment by hhyyrylainen** [comment]
It's possible to add text change suggestions after logging in and accepting the translator agreement on our translations website: https://translate.revolutionarygamesstudio.com/ (edit: though adding suggestions for `WIKI_` text is not really productive as that text is handled on our wiki and not with the translation tool).

Alternatively opening an issue here on github about the conflicting text is fine.
